install/scripts/update-database.php

```diff
@@ -0,0 +1,23 @@
+<?php
+// Include autoloader and table constants
```

Comment:
please add this to avoid calling this via web directly (as it does not feature any authentication whatsoever)

```php
// Check if we're in the CLI
if (@php_sapi_name() !== 'cli') {
	die('This script will only work in the shell.');
}
```